gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest.py.j2

```diff
@@ -247,6 +255,20 @@ class {{service.name}}RestTransport(_Base{{ service.name }}RestTransport):
             json_format.Parse(response.content, pb_resp, ignore_unknown_fields=True)
             {% endif %}{# method.lro #}
             resp = self._interceptor.post_{{ method.name|snake_case }}(resp)
+            if CLIENT_LOGGING_SUPPORTED and _LOGGER.isEnabledFor(logging.DEBUG):  # pragma: NO COVER
```

Comment:
We should likely log before any interceptors, too.

Reply:
FIled https://github.com/googleapis/gapic-generator-python/issues/2274